src/tactic/simp_result.lean

```diff
@@ -0,0 +1,130 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Gabriel Ebner, Scott Morrison
+-/
+import tactic.core
+
+/-!
+# simp_result
+
+`dsimp_result` and `simp_result` are a pair of tactics for
```

Comment:
Could you add another sentence or two up here explaining what you mean by "cleaning up"? As I scan through this PR/file, it's still a little bit mysterious what this is all for.

Reply:
@robertylewis, I added a paragraph of explanation, and pointed to the examples in the test file, which cover the substantial uses I envision so far.